gstreamer/src/pad.rs

```diff
@@ -116,15 +116,15 @@ pub trait PadExtManual {
     fn pull_range(&self, offset: u64, size: u32) -> Result<Buffer, FlowReturn>;
     fn get_range(&self, offset: u64, size: u32) -> Result<Buffer, FlowReturn>;
 
-    fn peer_query(&self, query: &mut QueryRef) -> bool;
-    fn query(&self, query: &mut QueryRef) -> bool;
+    fn peer_query(&self, query: &mut Query) -> bool;
+    fn query(&self, query: &mut Query) -> bool;
```

Comment:
With all these functions we now have the problem that it's a bug to pass a query in there that has a refcount > 1, but it's possible now. The query must be writable, which was the reason for passing a `&mut QueryRef` before (which guarantees exactly this).